.github/actions/prepare-ce-test-env/action.yml

```diff
@@ -48,11 +48,23 @@ runs:
       shell: bash
 
     - name: Install Tarantool
+      if: |
+        inputs.skip-tarantool-install == 'false' &&
+        !startsWith(inputs.tarantool-version, '3.')
       uses: tarantool/setup-tarantool@v1
-      if: ${{ inputs.skip-tarantool-install == 'false' }}
       with:
         tarantool-version: '${{ inputs.tarantool-version }}'
 
+    - name: Install Tarantool from repo
+      if: |
+        inputs.skip-tarantool-install == 'false' &&
+        startsWith(inputs.tarantool-version, '3.')
```

Comment:
I have two questions/suggestions:
- Do I understand correctly that in case of 3.0 the latest 3.0 release (alpha, beta or ...) will be installed?
- I suggest adding a comment that `curl ...` is a temporary trick until we can't use the `setup-tarantool` action, and it will be replaced in the future

Reply:
Answered in the comment in the action code.